gotestcover.go

```diff
@@ -45,13 +48,32 @@ func run() error {
 	if err != nil {
 		return err
 	}
+
+	coverage := 0.0
+	coveredPackages := 0
+	coverageRe := regexp.MustCompile(`coverage: (\d+\.\d+)%`)
+
 	cov := runAllPackageTests(ps, func(out string) {
 		fmt.Print(out)
+
+		if flagCoverageSummary {
+			if m := coverageRe.FindStringSubmatch(out); m != nil {
+				c, _ := strconv.ParseFloat(m[1], 32)
+				coverage += c
```

Comment:
You CAN'T sum floats: http://stackoverflow.com/questions/588004/is-floating-point-math-broken